packages/edit-site/src/utils/constants.js

```diff
@@ -1,5 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+// Navigation
+export const NAVIGATION_POST_TYPE = 'wp_navigation';
+
+// Templates.
+export const TEMPLATE_POST_TYPE = 'wp_template';
+export const TEMPLATE_PART_P0ST_TYPE = 'wp_template_part';
```

Comment:
Looks like there's a `0` in this one instead of an `O` 😁 

```suggestion
export const TEMPLATE_PART_POST_TYPE = 'wp_template_part';
```

Reply:
LOL - I blame my IDE

Reply:
Thanks for the lightening fast testing though! 🙇🏻